src/software/parameter/config/physics.yaml

```diff
@@ -0,0 +1,12 @@
+PhysicsConfig:
+  BallPhysicsConfig:
+    Restitution:
+      type: "double"
+      min: 0.0
+      max: 1.0
+      default: 0.8
+      description: The amount of energy retained when the ball bounces off objects
+    LinearDamping:
+      type: "double"
+      default: 0.2
+      description: The amount the ball speed slows down over time
```

Comment:
What was the logic behind making these dynamic params? If they have units we could maybe use them in our ball model, but I'm pretty sure they're unitless?

Reply:
This parameters seriously affect whether tests will pass or not, so if we're tuning them to the real world then we'll cause loads of tests to fail

Reply:
Are these going to be used for a "real" ball model as well, or just for simulation?

Reply:
only for simulation

Reply:
Hmmm not sure if DynamicParams are _really_ needed in this case, but I can look into changing it later.

Reply:
I'd really like to avoid adding more dynamic parameters here, we already have so many that it's difficult to go through if you don't know what you're looking for. Could we just make these constants that can be passed in?

Reply:
made these constants to pass in